static/app/views/insights/common/components/tableCells/starredTransactionCell.tsx

```diff
@@ -0,0 +1,77 @@
+import {useState} from 'react';
+
+import {Button} from 'sentry/components/core/button';
+import {IconStar} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {FlexContainer} from 'sentry/utils/discover/styles';
+import useProjects from 'sentry/utils/useProjects';
+import {
+  type StarTransactionParams,
+  useStarredTransactions,
+} from 'sentry/views/insights/common/utils/useStarredTransactions';
+
+interface Props {
+  containerProps?: React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  >;
```

Comment:
Is there a specific need for this? If not, would be good to take it out, to reduce the possibility of people futzing with this! I think re-styling table cells is not a great pattern, if the innards need to rendered in a different context it'd be better to expose the contents as their own component

Reply:
No, there was a copy and paste thing, sorry about that!